superduper/base/event.py

```diff
@@ -120,6 +120,7 @@ def execute(self, db: 'Datalayer'):
         """Execute the create event."""
         # TODO decide where to assign version
         db.metadata.create_component(self.component)
+        # TODO - do we really need to load the whole component?
         component = db.load(uuid=self.component['uuid'])
```

Comment:
We only need, uuid, dependencies which is available in the self.component itself

I think this db.load is putting component to cache, that the only relevant job its doing.

Reply:
To be resolved later.